tests/tests/test-proxy/test-proxy.ts

```diff
@@ -94,7 +94,7 @@ describeDevMoonbeam("Pallet proxy - shouldn't accept removed proxy", (context) =
           alith,
           context.polkadotApi.tx.proxy.removeProxy(baltathar.address, "Any", 0)
         );
-        expect(events2[4].method).to.be.eq("ExtrinsicSuccess");
+        expect(events2[5].method).to.be.eq("ExtrinsicSuccess");
```

Comment:
Why is this one changed ? This looks like a breaking change

Reply:
We've had to change these indices frequently, often because the ordering of the array changes (a new event is emitted, events are split into multiple ones, are combined, etc.)

I think we need a better way to express these expectations, something like "ExtrinsicSuccess" is followed by "SomeDepositEvent" but not "any error" would be awesome.

I'll try to see what caused this particular case...